.github/workflows/build.yml

```diff
@@ -42,10 +42,10 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Set up Ruby 3.0
+      - name: Set up Ruby 3.2.2
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '3.0'
+          ruby-version: '3.2.2'
```

Comment:
should be pretty stable if we just define version `3.2` here ... that would make sure we're getting minor version upgrades automatically